app/views/users/_form.html.erb

```diff
@@ -48,7 +48,12 @@
       </br>
       <%= field_set_tag _("General") do %>
         <%= checkbox_f f, :mail_enabled %>
+        <div class='col-md-4'>
+            <%= link_to_function(_("Test email"), "test_mail(this, '#{f.object.id}')",
+                                                :title => _("Sends an email for testing the configurations"), :id => "test_mail_button", :class =>"btn btn-success") + image_tag('/assets/spinner.gif', :id => 'test_indicator', :class => 'hide').html_safe if action_name == "edit" %>
```

Comment:
"Send a test message to the user's email address to confirm the configuration is working." ?